tests/unit/lib/sync/test_sync_flow.py

```diff
@@ -1,4 +1,7 @@
 from pathlib import Path
+
```

Comment:
Let's remove line 2 to 4, seems like an unneeded auto-import from IDE

Reply:
Good catch, addressed!